cmd/oras/copy.go

```diff
@@ -124,6 +132,7 @@ func runCopy(opts copyOptions) error {
 			copyOptions := oras.CopyOptions{
 				CopyGraphOptions: extendedCopyOptions.CopyGraphOptions,
 			}
+			copyOptions.WithTargetPlatform(targetPlatform)
```

Comment:
```suggestion
			if targetPlatform != nil {
				copyOptions.WithTargetPlatform(targetPlatform)
			}
```

Reply:
What does `nil` represent?

Reply:
Yeah `nil` should be considered as a `no input`. We should fix this in `oras-go`.

Reply:
Issue created: https://github.com/oras-project/oras-go/issues/318